scripts/gendepgraph.sh

```diff
@@ -6,4 +6,5 @@ if [ -n "$1" ]; then
 	out=$1
```

Comment:
Do we really want the dependency graph to be present as an image file at `/`? This would mean that every time we refactor something, we'd need to update this file as well. It'd be nice if we could leave this to the user and add a note in the readme about the function of the scripts present in `scripts/`

Reply:
The default is to put it in `docs`.  I had it as an image in the readme since I've seen that in a lot of Go projects.